tests/e2e/test_rosacli_cluster.go

```diff
@@ -854,6 +855,126 @@ var _ = Describe("Classic cluster creation validation",
 				Expect(err).NotTo(BeNil())
 				Expect(errorOutput.String()).To(ContainSubstring("etcd encryption cannot be disabled on clusters with FIPS mode"))
 			})
+
+		It("Create rosa cluster with additional security groups will work well via rosacli - [id:68971]",
```

Comment:
It's a day1-post case. Not day1-supplemental. And this option had been supported in day1

Reply:
Thanks @xueli181114, i did update the label can you please take a look again?

Reply:
@AkashKanni  I think we made things wrong. The title of the case is ```Create rosa cluster with additional security groups will validate well via rosacli```  not ```Create rosa cluster with additional security groups will work well via rosacli``` . It's a validation case of day1-negative.

Reply:
@xueli181114 i updated case title and label, could you PTAL again?